python/practices/practice-3/src/path1/triple_double_letters.py

```diff
@@ -0,0 +1,64 @@
+"""
+Check and prints words containing three consecutive double letters
+"""
+
+
+import os
+from helpers.file_reader import read_file_content
+from constant.constant_file import word_file
+
+
+def contains_three_double(word):
+    """
+    Check if a word contains three consecutive double letters.
+
+    Parameters:
+        - word (str): The input word to check.
+
+    Returns:
+        - bool: True if the word contains three consecutive double letters,
+        False otherwise.
+    """
+
+    index = 0
+
+    while index < len(word) - 5:
+        if (
+            word[index] == word[index + 1]
+            and word[index + 2] == word[index + 3]
+            and word[index + 4] == word[index + 5]
+        ):
+            return True
+        index += 1
+
+    return False
+
+
+def find_triple_double(file_content):
+    """
+    Prints words with triple double letters from word list.
+
+    Parameters:
+        - file_content (str): The content of the word list.
+
+    Returns:
+        None
+    """
+
+    if file_content:
```

Comment:
Please review the Writing Idiomatic Python 3 for If Statements.
Do not nest multiple if statements inside each other.